env.yml

```diff
@@ -29,19 +30,19 @@ dependencies:
 
   # ML packages
   - cuda-version # works also with CPU-only system.
-  - pytorch >=1.12
+  - pytorch <2.5
   - lightning >=2.0
   - torchmetrics
   - ogb
   - pytorch_geometric >=2.0 # Use `pyg` for Windows instead of `pytorch_geometric`
-  - wandb
+  - wandb >=0.18.5
   - mup
   - pytorch_sparse >=0.6
   - pytorch_cluster >=1.5
   - pytorch_scatter >=2.0
 
   # chemistry
-  - rdkit == 2024.03.4
+  - rdkit=2024.03.4
```

Comment:
Too strict. Can you do `<=` or `>=`?

Reply:
I'll run a test build with `<=` and report back.